src/frontends/tensorflow_common/src/op/sparse_fill_empty_rows.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/sparse_fill_empty_rows.hpp"
+
+#include "common_op_table.hpp"
+#include "openvino/op/convert.hpp"
+#include "utils.hpp"
+
+namespace ov::frontend::tensorflow::op {
+NamedOutputVector translate_sparse_fill_empty_rows_op(const NodeContext& node) {
+    default_op_checks(node, 4, {"SparseFillEmptyRows"});
+    auto indices = node.get_input(0);
+    auto values = node.get_input(1);
+    auto dense_shape = node.get_input(2);
+    auto default_value = node.get_input(3);
+    TENSORFLOW_OP_VALIDATION(node,
+                             values.get_element_type() != element::string,
+                             "TensorFlow Frontend does not support string inputs for SparseFillEmptyRows.");
+
+    auto indices_type = indices.get_element_type();
+    if (indices_type != element::i32 && indices_type != element::i64) {
+        indices = std::make_shared<ov::op::v0::Convert>(indices, element::i64);
+        indices_type = element::i64;
+    }
```

Comment:
it is not possible to have non `int64_t` indices_type by TF specification. As rule, (I know for several years) TF specification is correct. So we should expect only int64 type, otherwise, the model is incorrect.
So let us assume that it is of int64 type and check is not needed.

Reply:
Removed.